tasks/main.yml

```diff
@@ -0,0 +1,8 @@
+# role tasks
+- name: Provision VM (OVF)
+  include_tasks: deploy_ovf.yml
+  with_items: "{{ vmware_provision_OVFs }}"
+
+- name: Provision VM (CUSTOM)
+  include_tasks: manage_vm.yml
```

Comment:
If that one is "deploy_ovf.yml" then why is this one "manage_vm.yml" instead of "deploy_custom.yml" or similar?